.github/workflows/merge-up.yml

```
@@ -27,5 +27,5 @@ jobs:
         uses: alcaeus/automatic-merge-up-action@main
         with:
           ref: ${{ github.ref_name }}
-          branchNamePattern: '<major>.<minor>'
+          branchNamePattern: '<major>.x'
```

If we want to have a similar pattern as we do in PHPLIB and PHPC, this won't work.

I'd assume we continue to have branches for stable versions (e.g. 4.8, 5.0, etc.) and development branches for major verisons (e.g. 4.x, 5.x). In that case, the config should be:

```suggestion
          branchNamePattern: '<major>.<minor>'
          devBranchNamePattern: '<major>.x'
```

---

Oh, I reread 2 times the description of your other PRs but did not realize the property was renamed. My bad.